Northstar.Client/mod/resource/northstar_client_localisation_spanish.txt

```diff
@@ -85,11 +85,11 @@ Presiona Sí al estar de acuerdo. Esta opcion se puede cambiar en el menú de mo
 		"aitdm_archer_grunts" "Soldado Archer"
 
 		// northstar.custom localisation is just deciding not to work, so putting it here for now
-		"PL_sbox" "Sandbox"
-		"PL_sbox_lobby" "Vestíbulo de Sandbox"
-		"PL_sbox_desc" "como gmod pero peor"
+		"PL_sbox" "Custom"
```

Comment:
This change feels a bit odd tbh. Maybe `Sandbox` gamemodes are commonly called `Custom` in Spanish for some reason?

Reply:
Eh, I'm just gonna resolve this and assume it's fine. If it's not someone will probably step in and fix it again anyway ^^